apps/test/unit/craft/i18nTest.js

```diff
@@ -0,0 +1,30 @@
+import {assert} from '../../util/deprecatedChai';
```

Comment:
Any reason not to use `reconfiguredChai`?

Reply:
no idea! I was just copying what's in the existing `craftTest`